text/0000-buildpacks-config.md

```diff
@@ -0,0 +1,219 @@
+# Meta
+[meta]: #meta
+- Name: (fill in the feature name: My Feature)
+- Start Date: (fill in today's date: YYYY-MM-DD)
+- Author(s): (Github usernames)
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes:
+    - [RFC#19](https://github.com/buildpacks/rfcs/blob/main/text/0019-project-descriptor.md)
+    - [RFC#54](https://github.com/buildpacks/rfcs/blob/main/text/0054-project-descriptor-schema.md)
+    - [RFC#54](https://github.com/buildpacks/rfcs/blob/main/text/0054-project-descriptor-schema.md)
+    - [RFC#80](https://github.com/buildpacks/rfcs/blob/main/text/0080-builder-key-in-project-descriptor.md)
+    - [RFC#84](https://github.com/buildpacks/rfcs/blob/main/text/0084-project-descriptor-domains.md)
+
+# Summary
+[summary]: #summary
+
+A configuration file intended for end-users to influence how a buildpack build occurs.
+
+# Definitions
+[definitions]: #definitions
+
+Make a list of the definitions that may be useful for those reviewing. Include phrases and words that buildpack authors or other interested parties may not be familiar with.
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+- What use cases does it support?
+- What is the expected outcome?
+
+# What it is
+[what-it-is]: #what-it-is
+<!--
+This provides a high level overview of the feature.
+
+- Define any new terminology.
+- Define the target persona: buildpack author, buildpack user, platform operator, platform implementor, and/or project contributor.
+- Explaining the feature largely in terms of examples.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to existing users and new users.
+-->
+
+`./.buildpacks.<ext>`
```

Comment:
I'm afraid that using `buildpacks` for something that is `pack`-specific will be confusing when it doesn't work on other buildpack platforms.

Reply:
The intent of this file is to be platform agnostic, in other words NOT pack specific.

Reply:
I added a lot more details. Give it another read and see if it's any clearer where I'm trying to go with this proposal.